src/pages/_app.tsx

```diff
@@ -1,16 +1,40 @@
 // import '@/styles/globals.css'
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import { DSProvider } from "@nypl/design-system-react-components";
+import {
+  DSProvider,
+  useFeedbackBox,
+} from "@nypl/design-system-react-components";
+import React from "react";
+
+const onSubmit = async (values) => {
+  const response = await fetch("/api/feedback", {
+    method: "POST",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(values),
+  });
```

Comment:
I think you may need to move the `onSubmit` function inside of `App` to implement error handling: https://nypl.github.io/nypl-design-system/reservoir/v2/?path=/docs/components-form-elements-feedbackbox--docs#error

This doesn't work on Vercel but it does locally. I _think_ it'll work on the QA instance once it's deploy, but until then, we should render the error message to patrons.